## 39% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 30
Expected to fail tests: 1
Passed tests: 20
```
```
IDs of failed tests: 727230,702809,702733,702777,702732,704613,702957,702851,703495,703086,702948,702894,703297,704615,703503,727229,702855,703202,703391,702786,702844,702843,702807,702731,702808,702845,702775,702958,703629,702841 
```
```
IDs of expected to fail tests: 727232 
```
<h3>Failed tests (30)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=fix/clean-qr-scanned-address&PR_ID=20699&APK_NAME=20699.apk&TR_CASE_IDS=727230,702809,702733,702777,702732,704613,702957,702851,703495,703086,702948,702894,703297,704615,703503,727229,702855,703202,703391,702786,702844,702843,702807,702731,702808,702845,702775,702958,703629,702841">Rerun failed tests</a></li><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603903">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:470: in test_1_1_chat_text_message_delete_push_disappear
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#381">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#281">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603926">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `ChatsTab` by `accessibility id`: `chats-stack-tab` </div><div>Device 1: Tap on found: ChatsTab </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:378: in test_1_1_chat_edit_message
    home.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#365">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#275">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603940">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td>

```
critical/chats/test_1_1_public_chats.py:405: in test_1_1_chat_send_image_save_and_share
    if not self.chat_2.chat_message_input.is_element_displayed():
../views/base_element.py:221: in is_element_displayed
    return self.wait_for_visibility_of_element(sec, ignored_exceptions=ignored_exceptions)
../views/base_element.py:145: in wait_for_visibility_of_element
    .until(expected_conditions.visibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:152: in _predicate
    return _element_if_visible(driver.find_element(*locator))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E   Stacktrace:
E   UnknownError: An unknown server-side error occurred while processing the command. Original error: 'POST /element' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
E       at UIA2Proxy.command (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/jsonwp-proxy/proxy.js:355:13)
E       at processTicksAndRejections (node:internal/process/task_queues:95:5)
E       at AndroidUiautomator2Driver.helpers.doFindElementOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/lib/commands/find.js:30:10)
E       at doFind (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:36:17)
E       at wrappedCondFn (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:137:14)
E       at spin (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:221:20)
E       at waitForCondition (/mnt/sauce/appium/appium-v2.0.0/node_modules/asyncbox/lib/asyncbox.js:238:10)
E       at AndroidUiautomator2Driver.implicitWaitForCondition (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/timeout.ts:139:12)
E       at AndroidUiautomator2Driver.findElOrEls (/mnt/sauce/appium/appium-v2.0.0/appium_home/linux/node_modules/appium-uiautomator2-driver/node_modules/appium-android-driver/lib/commands/find.js:60:5)
E       at AndroidUiautomator2Driver.findElOrElsWithProcessing (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:60:12)
E       at AndroidUiautomator2Driver.findElement (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/basedriver/commands/find.ts:75:12)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#379">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#278">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603901">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div><div>Device 1: Find `Button` by `xpath`: `//*[@content-desc='pins-count']//android.widget.TextView` </div></blockquote></p>

```
critical/chats/test_1_1_public_chats.py:198: in test_1_1_chat_pin_messages
    chat.pinned_messages_count.wait_for_element_text(text="2",
../views/base_element.py:190: in wait_for_element_text
    self.driver.fail(message if message else "`%s` is not equal to expected `%s` in %s sec" % (
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Pinned messages count is not 2 as expected!
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#217">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#170">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603948">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:173: in test_wallet_send_asset_from_drawer
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the senders wallet is 0.1861 but should be 0.186
E    Eth amount in the receivers wallet is 0.0121 but should be 0.0122
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 1:00 PM
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603947">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Button` by `accessibility id`: `top-bar` </div><div>Device 1: Find `Button` by `accessibility id`: `top-bar` </div></blockquote></p>

```
critical/test_wallet.py:147: in test_wallet_send_eth
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Sender balance is not updated on Etherscan, it is 0.0 but expected to be -0.0001
E    Receiver balance is not updated on Etherscan, it is 0.0026 but expected to be 0.0027
E    Eth amount in the receivers wallet is 0.0121 but should be 0.0122
E    Eth amount in the senders wallet is 0.1861 but should be 0.186
E    The last transaction is not listed in activity for the receiver, expected timestamp is Today 12:55 PM
```

<br/><br/></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603902">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603941">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603938">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603937">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603912">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/a18b922c883e4e1a9ab52de03efd8d95?auth=3812549788fda08860abc35f974570da">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b84f67f18d184fc1b8028c1e6bf4c53b?auth=8c850ad4545359cda0fef3e9d8d6d21d">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603913">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_group_chat.py:19: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(3)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603932">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:385: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603933">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td>

```
Test setup failed: activity_center/test_activity_center.py:385: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603914">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:834: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603931">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:834: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603911">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:834: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603922">test_community_leave</a>, id: 702845 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:834: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603944">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td>

```
Test setup failed: critical/chats/test_public_chat_browsing.py:834: in prepare_devices
    self.drivers, self.loop = create_shared_drivers(2)
base_test_case.py:328: in create_shared_drivers
    raise e
base_test_case.py:318: in create_shared_drivers
    test_suite_data.current_test.testruns[-1].jobs[drivers[i].session_id] = i + 1
 '_asyncio.Future' object has no attribute 'session_id'
```

<br/><br/></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603945">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Looking for chat: 'general' </div><div>Device 1: Click until `ChatMessageInput` by `accessibility id`: `chat-message-input` will be presented </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the url https://status.app/c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b702f5e103734609828a76c32fbf94f2?auth=53176fcf3f711b194a92fcb3a3eeff38#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603906">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `BrowserTab` by `accessibility id`: `browser-stack-tab` </div><div>Device 1: Tap on found: BrowserTab </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:115: in test_links_deep_links
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'token-gated community e2e' was not requested to join by the deep link status.app://c/G0IAAGS9TbI9SrSPbDPplfBUeBBOIWhFRhGIKVlxntAg5CSvsJw4jurb636UKo7Lm2gTUS_M#zQ3shm7cS2MX6K1iinjNhiw88g1WH6fbE8rvnMHBq4RhznkWc
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b702f5e103734609828a76c32fbf94f2?auth=53176fcf3f711b194a92fcb3a3eeff38#216">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603943">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><p><blockquote><div>Device 1: Swiping up </div><div># STEP: Check Status Inu community tags in the Discover communities screen</div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:300: in test_community_discovery
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community ' 0xUX' is not in the Discover Communities list.
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#354">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603907">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><p><blockquote><div>Device 2: Tap on found: LogInButton </div><div>## Signed in successfully!</div></blockquote></p>

```
activity_center/test_activity_center.py:176: in test_add_contact_field_validation
    self.loop.run_until_complete(run_in_parallel(((_device_1_creates_user, {}),
/usr/lib/python3.10/asyncio/base_events.py:649: in run_until_complete
    return future.result()
__init__.py:52: in run_in_parallel
    returns.append(await k)
/usr/lib/python3.10/concurrent/futures/thread.py:58: in run
    result = self.fn(*self.args, **self.kwargs)
activity_center/test_activity_center.py:166: in _device_1_creates_user
    self.profile_1.driver.reset()
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/extensions/applications.py:299: in reset
    self.execute(Command.RESET)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 A session is either terminated or not started
E   Stacktrace:
E   NoSuchDriverError: A session is either terminated or not started
E       at asyncHandler (/mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/protocol.js:315:15)
E       at /mnt/sauce/appium/appium-v2.0.0/packages/base-driver/lib/protocol/protocol.js:518:15
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/layer.js:95:5)
E       at next (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/route.js:144:13)
E       at Route.dispatch (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/route.js:114:3)
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/layer.js:95:5)
E       at /mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:284:15
E       at param (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:365:14)
E       at param (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:376:14)
E       at Function.process_params (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:421:3)
E       at next (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:280:10)
E       at logger (/mnt/sauce/appium/appium-v2.0.0/node_modules/morgan/index.js:144:5)
E       at Layer.handle [as handle_request] (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/layer.js:95:5)
E       at trim_prefix (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:328:13)
E       at /mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:286:9
E       at Function.process_params (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:346:12)
E       at next (/mnt/sauce/appium/appium-v2.0.0/node_modules/express/lib/router/index.js:280:10)
E       at /mnt/sauce/appium/appium-v2.0.0/node_modules/body-parser/lib/read.js:137:5
E       at AsyncResource.runInAsyncScope (node:async_hooks:204:9)
E       at invokeCallback (/mnt/sauce/appium/appium-v2.0.0/node_modules/raw-body/index.js:238:16)
E       at done (/mnt/sauce/appium/appium-v2.0.0/node_modules/raw-body/index.js:227:7)
E       at IncomingMessage.onEnd (/mnt/sauce/appium/appium-v2.0.0/node_modules/raw-body/index.js:287:7)
E       at IncomingMessage.emit (node:events:513:28)
E       at endReadableNT (node:internal/streams/readable:1359:12)
E       at processTicksAndRejections (node:internal/process/task_queues:82:21)
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8fb2380a283941e980575206d1b8b76e?auth=f8e9eb3849008dc361aace4e873332f1#133">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf62ce82e95e44cebb4dbd363f9cab9d?auth=bf3a297c0d6d465543d64757fc74ad70#200">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603925">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div># STEP: Device1 check that contact appeared in contact list mutually</div></blockquote></p>

```
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:703: in urlopen
    httplib_response = self._make_request(
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:449: in _make_request
    six.raise_from(e, None)
<string>:3: in raise_from
    ???
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:444: in _make_request
    httplib_response = conn.getresponse()
/usr/lib/python3.10/http/client.py:1375: in getresponse
    response.begin()
/usr/lib/python3.10/http/client.py:318: in begin
    version, status, reason = self._read_status()
/usr/lib/python3.10/http/client.py:287: in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
E   http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:
activity_center/test_activity_center.py:142: in test_activity_center_contact_request_accept_swipe_mark_all_as_read
    self.home_2.navigate_back_to_home_view()
../views/base_view.py:425: in navigate_back_to_home_view
    while not self.chat_floating_screen.is_element_disappeared(1) \
../views/base_element.py:231: in is_element_disappeared
    return self.wait_for_invisibility_of_element(sec)
../views/base_element.py:154: in wait_for_invisibility_of_element
    .until(expected_conditions.invisibility_of_element_located((self.by, self.locator)))
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:86: in until
    value = method(self._driver)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/support/expected_conditions.py:319: in _predicate
    target = driver.find_element(*target)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/webdriver.py:409: in find_element
    return self.execute(RemoteCommand.FIND_ELEMENT, {'using': by, 'value': value})['value']
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:343: in execute
    response = self.command_executor.execute(driver_command, params)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:291: in execute
    return self._request(command_info[0], url, body=data)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/remote_connection.py:312: in _request
    response = self._conn.request(method, url, body=body, headers=headers)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/request.py:78: in request
    return self.request_encode_body(
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/request.py:170: in request_encode_body
    return self.urlopen(method, url, **extra_kw)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/poolmanager.py:376: in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:787: in urlopen
    retries = retries.increment(
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/util/retry.py:550: in increment
    raise six.reraise(type(error), error, _stacktrace)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/packages/six.py:769: in reraise
    raise value.with_traceback(tb)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:703: in urlopen
    httplib_response = self._make_request(
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:449: in _make_request
    six.raise_from(e, None)
<string>:3: in raise_from
    ???
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/urllib3/connectionpool.py:444: in _make_request
    httplib_response = conn.getresponse()
/usr/lib/python3.10/http/client.py:1375: in getresponse
    response.begin()
/usr/lib/python3.10/http/client.py:318: in begin
    version, status, reason = self._read_status()
/usr/lib/python3.10/http/client.py:287: in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
 ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8fb2380a283941e980575206d1b8b76e?auth=f8e9eb3849008dc361aace4e873332f1#99">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf62ce82e95e44cebb4dbd363f9cab9d?auth=bf3a297c0d6d465543d64757fc74ad70#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603934">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603929">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603946">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603921">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603920">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for a message by text: Message AFTER edit 2 (Edited) </div><div>Device 2: Find `ChatElementByText` by `xpath`: `//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:378: in test_community_message_edit
    self.channel_2.set_reaction(message_text_after_edit)
../views/chat_view.py:1054: in set_reaction
    self.chat_element_by_text(message).long_press_until_element_is_shown(element)
../views/base_element.py:327: in long_press_until_element_is_shown
    element = self.find_element()
../views/chat_view.py:116: in find_element
    self.wait_for_visibility_of_element(20)
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 2: ChatElementByText by xpath:`//*[starts-with(@text,'Message AFTER edit 2 (Edited)')]/ancestor::android.view.ViewGroup[@content-desc='chat-item']` is not found on the screen after wait_for_visibility_of_element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#123">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603919">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td>

```
Test setup failed: base_test_case.py:370: in setup_method
    driver.execute_script("sauce:context=Started %s" % method.__name__)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:405: in execute_script
    return self.execute(command, {"script": script, "args": converted_args})["value"]
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py:345: in execute
    self.error_handler.check_response(response)
../../../../status-app-prs@tmp/venv/lib/python3.10/site-packages/appium/webdriver/errorhandler.py:122: in check_response
    raise exception_class(msg=message, stacktrace=format_stacktrace(stacktrace))
 session backend not connected
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (1)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603950">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `EditBox` by `accessibility id`: `add-address-to-watch` </div><div>Device 1: Type `0x8d2413447ff297d30bdc475f6d5cb00254685aae` to `EditBox` </div></blockquote></p>

```
critical/test_wallet.py:237: in test_wallet_add_remove_watch_only_account
    self.wallet_view.add_watch_only_account(address=address_to_watch, account_name=new_account_name)
../views/wallet_view.py:150: in add_watch_only_account
    self.account_has_activity_label.wait_for_visibility_of_element()
../views/base_element.py:147: in wait_for_visibility_of_element
    raise TimeoutException(
 Device 1: Text by accessibility id:`account-has-activity` is not found on the screen after wait_for_visibility_of_element 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9deb342c9d45519ec05d4f373faea9?auth=6b1b95aaf88c8f40d9f7d472f20ebb4d#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (20)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603908">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#147">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#121">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603915">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#316">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#244">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603905">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#258">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#194">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603900">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e298852dfc9347b486afd78a56abf08e?auth=4fe2e3ab7c18470c66b505308974610b#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6ea0b67c5cd34204b4bb8d36fe753a13?auth=ec1dddddae77a1873da5d8314fe881f6#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603930">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c286c07b8aac4c7dbd7f775bf9df23fa?auth=c5acde405a378e831c189339920bb6d3#128">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/6c36335ecbfd4bfdb9be1d007fe402a9?auth=e24ac46f6a0d8628714e2063cd081cc1#99">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603949">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bd9deb342c9d45519ec05d4f373faea9?auth=6b1b95aaf88c8f40d9f7d472f20ebb4d#38">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603935">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603904">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#76">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603928">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#105">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603923">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#52">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603939">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/4d0e0336563c4623bde6686ede3d3522?auth=872ad802da3761a1e979de87d9714c5e#124">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603924">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8fb2380a283941e980575206d1b8b76e?auth=f8e9eb3849008dc361aace4e873332f1#56">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bf62ce82e95e44cebb4dbd363f9cab9d?auth=bf3a297c0d6d465543d64757fc74ad70#55">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603910">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5827c84ccae41e6bd757839b2965eb0?auth=c35ec315c9a74a0c015f3c1400f19e4f#172">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c1c9bc7f97844439bb40a58b3608a59?auth=23312ec5bfd69c3c6efbae462e8064d3#129">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603909">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5827c84ccae41e6bd757839b2965eb0?auth=c35ec315c9a74a0c015f3c1400f19e4f#95">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c1c9bc7f97844439bb40a58b3608a59?auth=23312ec5bfd69c3c6efbae462e8064d3#84">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603942">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/d5827c84ccae41e6bd757839b2965eb0?auth=c35ec315c9a74a0c015f3c1400f19e4f#106">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7c1c9bc7f97844439bb40a58b3608a59?auth=23312ec5bfd69c3c6efbae462e8064d3#116">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603936">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#171">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603927">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#234">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603918">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#294">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603917">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3603916">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c8fee5bd87e04623a0a2bd34d899b6e2?auth=fda5a67ec120e39bb995f49c24baa33a">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/655c8cdf478c4722b1de85eac5f6baf6?auth=11364ebd8a1b68ad19ea3ad712ef868a#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/settings/wallet/saved_addresses/add_address_to_save/view.cljs
+++ b/src/status_im/contexts/settings/wallet/saved_addresses/add_address_to_save/view.cljs
@@ -163,9 +163,8 @@
                                                (rf/dispatch
                                                 [:open-modal :screen/settings.save-address]))
                                              [address ens-name? address-or-ens])]
-    (rn/use-mount (fn []
-                    (rf/dispatch [:wallet/clean-scanned-address])
-                    (rf/dispatch [:wallet/clear-address-to-save])))
+    (rn/use-unmount #(rf/dispatch [:wallet/clean-scanned-address]))
+    (rn/use-mount #(rf/dispatch [:wallet/clear-address-to-save]))
     [quo/overlay {:type :shell}
      [floating-button-page/view
       {:footer-container-padding 0

